modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -330,6 +333,14 @@ public void lastApplied(long lastAppliedIndex, long lastAppliedTerm) throws Stor
         });
     }
 
+    private void lastAppliedBusy(AbstractWriteBatch writeBatch, long lastAppliedIndex, long lastAppliedTerm) throws RocksDBException {
```

Comment:
Why is this method called like that? Can we rename it to `saveLastApplied`?

Reply:
We can, but why? is there a need for this?

Reply:
I think yes, current name does a poor job of describing what this method actually does

Reply:
Fix it